terragrunt/aws/cloudfront/s3.tf

```
@@ -1,5 +1,5 @@
 module "log_bucket" {
-  source            = "github.com/cds-snc/terraform-modules?ref=v5.1.8//S3_log_bucket"
+  source            = "github.com/cds-snc/terraform-modules?ref=v7.3.2//S3"
```

Minor typo here - it should be:
```suggestion
  source            = "github.com/cds-snc/terraform-modules?ref=v7.3.2//S3_log_bucket"
```